README.md

```diff
@@ -1,46 +1,62 @@
-# Getting Started with Create React App and Redux
+# Calculator
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+This app implements a simple calculator. This is an exercise in design and development, using React, Redux and CSS in JS.
 
-## Available Scripts
+## Features
 
-In the project directory, you can run:
+The app contains three pages:
 
-### `npm start`
+### Calculator (home page)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+Composed of a thin strip (aprox. 280px) with variable height. It is composed of different section.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+#### Keys (tabbable)
 
-### `npm test`
+The numbers and opperations dispatches write commands to the store. The **clear** key dispatches a delete all command, and the **back** key dispatches a delete command for the last digit. The **save** key register dispatches a save command with the current expression, so it can appear in the Scope and in the History pages.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+The Keys group is accessible by tabbing. In fact, they are divided into two distinct tabbable groups, one containing 0-9, period and basic arythmetic operations, and another containing the other keys. The ARROWS can be used to select a specific key, including moveing from one group to the other (and even to other components, as scope and navigation). Once a key is focused, it can be activated by the SPACEBAR or ENTER keys. The ESC key activates the clear key.
```

Comment:
```suggestion
The Keys group is accessible by tabbing. In fact, they are divided into two distinct tabbable groups, one containing 0-9, period and basic arythmetic operations, and another containing the other keys. The ARROWS can be used to select a specific key, including moving from one group to the other (and even to other components, as scope and navigation). Once a key is focused, it can be activated by the SPACEBAR or ENTER keys. The ESC key activates the clear key.
```